integrations/unity-csharp/README.md

```diff
@@ -0,0 +1,115 @@
+Vault SDK Version:     0.5.0
+Toolsets:              Requires Unity >=2019.3
+```
+
+## Resources Required
+Unity Postprocessing >=v2.3.0
+
+## How to use the sample
+<!-- TODO: Explain how this sample can be used and what is required to get it running -->
+There are two projects included:
+
+* The basic package, including a basic flight camera and collider, this demonstrates the basic concepts required to get simulations and games using VDK running in unity. This is ideal for developers wishing to use VDK with their own projects
+* The advanced project: includes a driving simulator and third person ragdoll. This demonstrates use of VDK with Unity physics and is good for exploring and testing interactions of unity systems with Unlimited Detail point clouds.
+
+**Both examples require a valid license for Euclideon Vault SDK, trial licenses can be obtained [here](https://zfrmz.com/gwVUru84d60yUedxmLx9/?ref=Unity%20Sample%20Code)** 
+
+### Installation
+
+Both examples are tested with Unity 2019.3.0f6, all steps must be repeated for each package in their respective directories.
+
+1. Download and extract VDK 0.5.0 package from [here](https://earth.vault.euclideon.com) using your license credentials (if you do not have one, free trials are available from [here](https://zfrmz.com/gwVUru84d60yUedxmLx9/?ref=Unity%20Sample%20Code) )
+2. Download the Unity VDK example from [here]()
+3. Copy the files from Euclideon_vdk0.5.0/lib/(_your operating system here_)/to Assets/VDK in your Unity project
+4. Replace the username and password lines from _Assets/VDK/vdkLogin.cs_ to your vault login details **YOUR UNITY EDITOR MAY CRASH ON LOAD IF THIS STEP IS NOT COMPLETED**
+5. Open Unity Hub, Select Add and specify the downloaded project as the source folder
+6. Open Window \-> package manager \-> Post Processing \-> update (ensure that postprocessing packages are installed and up to date, \>)
+
+
+
+
+
+
+## Basic Example
+
+This is an example demonstrating how to use VDK with unity, it includes a minimalist example of a flight camera and an attached collider. 
+Unlimited detail rendering is implemented as a postprocessing effect that can be applied to cameras in order to display UDS objects.
+
+### Sample Scene Structure
+![The Structure of the basic sample scene](./docs/sampleSceneStructure.png "Structure of the Sample Scene")
+
+#### Main Camera
+
+The main view used in the example. It has a flight camera script attached to it to enable user interation. Of importance to this object is 
+the implemented post process layer and volume properties which must be included for the camera to view UDS files. 
+
+![Sample UDS camera Settings](./docs/sampleCameraSettings.png "Sample UDS Camera Settings")
+
+
+### UDS Model
+
+Each of UDS to be loaded in unity is represented as a one of these models.
+
+### vdkLogin
+
+This file contains the login logic for the unity example, including login credentials. GlobalVDKContext contains a [vdkContext]() for managing licensing 
```

Comment:
It's kind of strange having `vdkContext` as a link to nowhere, consider using a back tick, `, instead